demos/funnyThingsApp/composeGui.yml

```diff
@@ -12,6 +12,7 @@ x-yarp-base: &yarp-base
     - "${XAUTHORITY}:/root/.Xauthority"
     - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
     - "/dev:/dev"
+    - "${HOME}:/root"
```

Comment:
This is not what was required. This should be the FILE_INPUT_PATH as done [here](https://github.com/icub-tech-iit/appsAway/blob/5eebb8b24d7b81d686cb2b6dad7bc3ddab5c9c17/demos/googleSpeechApp/main.yml#L22) 

And the GUI of the deployment should be modified to accept the path from the user as [here](https://github.com/icub-tech-iit/appsAway/blob/master/demos/googleSpeechApp/gui/gui_conf.ini)